Backend/.env

```
@@ -1 +1,7 @@
-Mongo_URL = mongodb+srv://shanmukhapadmakumarchalla_db_user:PaihfMvjeoHJ6b76@cluster0.s6yu20q.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0
\ No newline at end of file
+Mongo_URL = mongodb+srv://shanmukhapadmakumarchalla_db_user:PaihfMvjeoHJ6b76@cluster0.s6yu20q.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0
```

dotenv-linter (3.3.0)

[warning] [LowercaseKey] The Mongo_URL  key should be in uppercase

(LowercaseKey)